apps/web-login/CallBack.html

```diff
@@ -0,0 +1,53 @@
+<!-- 네이버 callback 전용 html 파일 -->
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <script
+      type="text/javascript"
+      src="https://static.nid.naver.com/js/naverLogin_implicit-1.0.3.js"
+      charset="utf-8"
+    ></script>
```

Comment:
이렇게 내부에 아무것도 없는 경우 다음처럼 쓸 수 있습니다.
```suggestion
    <script
      type="text/javascript"
      src="https://static.nid.naver.com/js/naverLogin_implicit-1.0.3.js"
      charset="utf-8"
    />
```